server/controllers/finance/reports/priceList/report.handlebars

```diff
@@ -0,0 +1,49 @@
+{{> head title="{{translate 'FORM.LABELS.PRICE_LIST' }}" }}
```

Comment:
This template must be updated to be conform with the `tarification` model, you can remove columns like `consumable` and `default quantity`

![price list correct](https://user-images.githubusercontent.com/5445251/41742837-1881425a-7597-11e8-94ee-bbb917a35b3d.PNG)